app/views/users/show.html.erb

```diff
@@ -1,10 +1,41 @@
-<!DOCTYPE html>
-<html>
-<head>
-  <title>I am user show</title>
-</head>
-<body>
+<section class="container">
+<div class='d-flex mr-4'>
+<div class="card m-5 text-decoration-none mr-4" style="width: 15rem; heigth: 15rem;">
+      <% if @user.photo %>
+        <img src="<%= @user.photo %>" class="rounded float-start" alt="...">
+      <% else %>
+          <p> No image found </p>
+      <% end %>
+</div>
+  <div class="card mt-5 mb-5 --bs-dark bg-success border border-dark-subtle" style="width: 60rem;">
+  <div class="card-body ">
+    <h1 class="card-title text-light text-decoration-none"><%= @user.name %></h1>  
+     <h6 class="card-text text-light text-end">Number of posts: <span><%= @user.PostsCounter %></span></h6>
+  </div>
+    </div>
+    </div>
 
 
-</body>
-</html>
\ No newline at end of file
+
+
+  <div class="card mt-5 text-light mb-5 --bs-dark bg-secondary border border-dark-subtle" style="width: 80rem;">
+    <div class='card-body'>
+      <h2>Bio: </h2>
+      <h6><%= @user.bio %></h6>
+    </div>
+  </div>
+  
+  <% counter = 1 %>
+  <% @user.posts.each do |post| %>
+  <div class="card mt-5 mb-5 --bs-dark bg-warning border border-dark-subtle" style="width: 80rem;">
+    <div class='card-body'>
+      <h3 class="card-title text-black">Post #<%= counter %></h3>
+      <h6><%= post.text %></h6>
+      <%= link_to 'See all posts', user_posts_path(@user), class: "btn btn-secondary mt-3"%>
+      <h6 class='card-text text-black text-end'>Comments: <%= post.CommentsCounter%> Likes: <%= post.LikesCounter %></h6>
+   </div>
+  </div>
+    <% counter += 1 %>
```

Comment:
- [ ] Good job working on this but please ensure that you are only displaying the recent posts using the `recent_posts` method created in the User model